examples/speech_to_text/visualize.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import argparse
+import pprint
+
+
+def read_scores_from_folder(folder_path):
+    score_file_path = os.path.join(folder_path, "scores.tsv")
+    if os.path.isfile(score_file_path):
+        with open(score_file_path, "r") as f:
+            contents = [line.strip() for line in f.read().split("\n") if line.strip()]
+        return contents
+    else:
+        return None
+
+
+def read_scores_files(output_folder):
+    all_contents = []
+
+    if not os.path.isdir(output_folder):
+        raise ValueError("Output folder does not exist")
+
+    output_folder = os.path.abspath(output_folder)
+
+    for folder in os.listdir(output_folder):
+        folder_path = os.path.join(output_folder, folder)
+
+        if os.path.isdir(folder_path):
+            contents = read_scores_from_folder(folder_path)
+            if contents:
+                all_contents.append(contents)
+    return all_contents
+
+
+def process_result(output_folder, metric_names):
```

Comment:
Later you check for `metric_names is None`, so maybe you want to write this as
```suggestion
def process_result(output_folder, metric_names=None):
```